packages/core/src/command/hl7v2-subscriptions/__tests__/hl7v2-roster-generator.test.ts

```diff
@@ -12,6 +13,53 @@ describe("AdtRosterGenerator", () => {
   let patientId: string;
   let cxId: string;
   let scrambledId: string;
+
+  const mockMinimalSchema = {
+    scrambledId: "ID",
+    firstName: "FIRST NAME",
+    lastName: "LAST NAME",
+    dob: "DOB",
+  };
+
+  const mockSchema = {
+    ...mockMinimalSchema,
+    genderAtBirth: "GENDER",
+    address1AddressLine1: "STREET ADDRESS", // TODO: remove this
```

Comment:
nit: TODOs should always be tied to an issue number, or removed - nobody is going to remember what this is here for, or why